bedrock/firefox/views.py

```diff
@@ -284,7 +284,7 @@ class LatestFxView(TemplateView):
     """
     non_fx_redirect = 'firefox.new'
 
-    @vary_on_headers('User-Agent')
+    @cache_control(max_age=0)
```

Comment:
NIT: why not use the `never_cache` decorator instead. IIRC they should have the same result, but it's a bit more explicit.